GVFS/GVFS.FunctionalTests/Tools/GitHelpers.cs

```diff
@@ -27,7 +28,19 @@ private static string LockHolderCommandPath
         {
             get
             {
-                return Path.Combine(Settings.Default.CurrentDirectory, LockHolderCommand);
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))

```

Comment:
Can you leave a comment here about why we need this change? (the differences between Windows and Mac and how we run the functional tests)